tests/microservices/Microservices.IsIdentifiable.Tests/RunnerTests/DicomFileRunnerTest.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Microservices.IsIdentifiable.Options;
+using Microservices.IsIdentifiable.Reporting;
+using Microservices.IsIdentifiable.Reporting.Reports;
+using Microservices.IsIdentifiable.Runners;
+using NUnit.Framework;
+using Smi.Common.Options;
+using Smi.Common.Tests;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using System.Linq;
+
+
+namespace Microservices.IsIdentifiable.Tests.RunnerTests
+{
+    public class DicomFileRunnerTest
+    {
+        #region Fixture Methods
+
+        private const string DataDirectory = @"../../../../../../../data/";
+        private DirectoryInfo _tessDir;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            TestLogger.Setup();
+
+            GlobalOptions globals = new GlobalOptionsFactory().Load();
+
+            var testRulesDir = new DirectoryInfo(Path.Combine(TestContext.CurrentContext.TestDirectory, "data", "IsIdentifiableRules"));
+            testRulesDir.Create();
+            globals.IsIdentifiableOptions.DataDirectory = testRulesDir.Parent.FullName;
+            _tessDir = new DirectoryInfo(Path.Combine(testRulesDir.Parent.FullName, "tessdata"));
+            _tessDir.Create();
+            string dest = Path.Combine(_tessDir.FullName, "eng.traineddata");
+            if (!File.Exists(dest))
+                File.Copy(Path.Combine(DataDirectory, "tessdata", "eng.traineddata"), dest);
+
+        }
+
+        [OneTimeTearDown]
+        public void OneTimeTearDown() { }
+
+        #endregion
+
+        #region Test Methods
+
+        [SetUp]
+        public void SetUp() { }
+
+        [TearDown]
+        public void TearDown() { }
+
+        #endregion
+
+        #region Tests
+
+        [Test]
```

Comment:
We should probably add the true/false test cases for this (IgnoreTextLessThan x).  Unless that test is elsewhere.

```
[TestCase(true)]
[TestCase(false)]
public void IgnorePixelDataLessThan(bool ignoreShortText)
```